tests/common/plugins/sanity_check/__init__.py

```diff
@@ -13,6 +13,14 @@
 logger = logging.getLogger(__name__)
 
 
+def pytest_addoption(parser):
```

Comment:
I get the rest of the changes, but why do these need to be moved?

Reply:
This is reverting #1642.

Reply:
Ok, this seems like an odd place for these, let's revisit it later. :P